university-task-02/src/pages/Common/Footer/Footer.jsx

```diff
@@ -0,0 +1,48 @@
+import FooterBackground from '../../../assets/img/Footer-Background.svg';
+import { BUTTONS, SOCIALLINKS } from '../../../constants/constants';
+import Column from '../Column/Column';
+import './Footer.css';
+import Img from '../Img/Img';
+import SocialLink from '../SocialLinks/SocialLinks';
+
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+    <Img className={"footer-background"} src = {FooterBackground} />
+    <div className="footer-content">
+      <div className="footer-column-left">
+        <Img  
+          src={require("../../../assets/img/Nav-logo.svg").default} 
```

Comment:
Wrong approach for image's imp[ort. You can just import file like 
``` import logo from '.../assets/logo.svg'```
and use this variable as a src attribute value
```<img src={logo} />```